src/logic/operate.js

```diff
@@ -11,6 +11,9 @@ const operate = (numberOne, numberTwo, operation) => {
     case 'X':
       return first.times(second).toString();
     case '÷':
+      if (numberTwo === '0') {
+        return '0';
+      }
```

Comment:
- [ ] <b>OPTIONAL:</b> This is mathematically wrong. any number divide by 0  should be `INFINITY` not 0. since this project is about writing tests, I won't be marking this as `REQUIRED` but you should consider fixing that.